lib/fe.ex

```diff
@@ -15,4 +15,23 @@ defmodule FE do
   """
   @spec const(a) :: (any -> a) when a: var
   def const(a), do: fn _ -> a end
+  
+  @doc """
+  Transforms a n-ary function in an unary one that accepts its first
+  argument and returns a function expecting the next one, successively
```

Comment:
Would you mind adding a few `doctest` examples here?

I admit I had some difficulty with understanding what the function does on the first reading of this description.

How about something along these lines?

```
Transforms a n-ary function f(x1, x2, ..., xn) to a unary function that accepts the f's first argument x1 and returns the unary function that accepts x2, and so on until xn.
The unary function that accepts xn returns the same value as f(x1, x2, ..., xn).
```